narwhal/test-utils/src/lib.rs

```diff
@@ -764,19 +766,29 @@ impl<R> Builder<R> {
         self
     }
 
+    pub fn stake_distribution(mut self, stake: VecDeque<Stake>) -> Self {
+        self.stake = stake;
+        self
+    }
+
     pub fn rng<N: rand::RngCore + rand::CryptoRng>(self, rng: N) -> Builder<N> {
         Builder {
             rng,
             epoch: self.epoch,
             committee_size: self.committee_size,
             number_of_workers: self.number_of_workers,
             randomize_ports: self.randomize_ports,
+            stake: self.stake,
         }
     }
 }
 
 impl<R: rand::RngCore + rand::CryptoRng> Builder<R> {
     pub fn build(mut self) -> CommitteeFixture {
+        if !self.stake.is_empty() {
```

Comment:
Maybe initialize self.stake to a vec deque of 1 when it is empty?

Reply:
Yeah thought about this, but given this check I considered quite safe to just fallback to `1` if there are no more elements in the vec (practically it should be empty in this case)

Reply:
The intention was to simplify the usages of `self.stake` so you don't need to `unwrap_or_(1)`, but this is minor.